create-release.py

```diff
@@ -239,7 +239,7 @@ def update_version_to_dev() -> None:
 
         sed(_source('packages/theme/src/index.ts'),
             r"https://elyra.readthedocs.io/en/stable/",
-            f"https://elyra.readthedocs.io/en/latest/")
+            f"https://elyra.readthedocs.io/en/v{new_version}/")
```

Comment:
You need to consider dev->release and then release->dev and this is only handling on side of it.